src/EnergyPlus/CrossVentMgr.cc

```diff
@@ -569,11 +577,11 @@ namespace CrossVentMgr {
 
 		// Calculate the opening area for all apertures
 		for ( Ctd = 1; Ctd <= AirflowNetworkSurfaceUCSDCV( 0, ZoneNum ); ++Ctd ) {
-			CompNum = AirflowNetworkLinkageData( Ctd ).CompNum;
-			TypeNum = AirflowNetworkCompData( CompNum ).TypeNum;
-			if ( AirflowNetworkCompData( CompNum ).CompTypeNum == CompTypeNum_DOP ) {
+			EvolveParaUCSDCV_CompNum = AirflowNetworkLinkageData( Ctd ).CompNum;
```

Comment:
@JasonGlazer If I'm following correctly, CompNum and TypeNum are only used inside this for loop.  So they could be declared within the loop and don't need to be in the anonymous namespace nor in clear_state